public/app/plugins/datasource/loki/querybuilder/components/QueryPatternsModal.tsx

```diff
@@ -66,7 +66,17 @@ export const QueryPatternsModal = (props: Props) => {
   };
 
   return (
-    <Modal isOpen={isOpen} title="Kick start your query" onDismiss={onClose}>
+    <Modal
+      isOpen={isOpen}
+      title="Kick start your query"
+      onDismiss={onClose}
+      className={css`
+        width: 85vw;
+        @media (max-width: 768px) {
```

Comment:
I'd check other places where we use breakpoints specified in theme: https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker.tsx#LL237. Here is how breakpoints work: https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/themes/breakpoints.ts#L24.

Reply:
i didn't know this existed, thank you!